src/styles/ulils/__mixins.scss

```
@@ -0,0 +1,4 @@
+@mixin square-size-day {
```

looks like you can use it just for day

```suggestion
@mixin square {
```